addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/VerificationDetectionScanRule.java

```diff
@@ -41,6 +43,21 @@ public class VerificationDetectionScanRule extends PluginPassiveScanner {
     private static final VerificationComparator COMPARATOR =
             VerificationRequestDetails.getComparator();
 
+    static final List<String> COMMON_LOGOUT_STRINGS =
```

Comment:
Do you have real examples for all the values?

Reply:
I'd prefer we support known use cases not add more than it should and end up excluding a good verification.